https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/7758240c79a3ed15459da5d150a0b1c3612e3f79

From 7758240c79a3ed15459da5d150a0b1c3612e3f79 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Thu, 15 Dec 2022 12:20:40 +0100
Subject: [PATCH] bluez: fix compilation without ldac_abr

The define for this constant was removed but it was LDACBT_EQMID_MQ
before.
--- a/spa/plugins/bluez5/a2dp-codec-ldac.c
+++ b/spa/plugins/bluez5/a2dp-codec-ldac.c
@@ -243,7 +243,7 @@ static int codec_reduce_bitpool(void *data)
 #else
 	struct impl *this = data;
 	int res;
-	if (this->eqmid == LDACBT_EQMID_BITRATE_330000 || !this->enable_abr)
+	if (this->eqmid == LDACBT_EQMID_MQ || !this->enable_abr)
 		return this->eqmid;
 	res = ldacBT_alter_eqmid_priority(this->ldac, LDACBT_EQMID_INC_CONNECTION);
 	return res;
-- 
GitLab
